ironfish/src/rpc/routes/accounts/rescanAccount.ts

```diff
@@ -35,6 +35,7 @@ router.register<typeof RescanAccountRequestSchema, RescanAccountResponse>(
       if (request.data.reset) {
         await node.accounts.reset()
       }
+      await node.accounts.updateHead()
```

Comment:
Rather than calling `updateHead` here, I think we should update `scanTransactions` to set `this.headHash` to `chain.head` if `this.headHash` is null before the scan begins.